### PR TITLE
Issue 20669 fix old dot assets title

### PR DESCRIPTION
**Pull request by nollymar**
These changes fix existing dotAssets' title. The identifier is replaced by the file name

### CODE REVIEW

**Review by dsilvam — APPROVED** [review]

**Comment by dsilvam** [comment]
@nollymar can you please add the description in this PR to the UT's javadoc

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210719CleanUpDotAssetTitle.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        + java.io.File.separator
+                        + inode.charAt(1)
+                        + java.io.File.separator
+                        + inode
+                        + java.io.File.separator
+                        + dotAsset.get("velocity_var_name"));
+                final File[] files = binaryFileFolder.listFiles();
+
+                if (null != files && files.length > 0){
+                    new DotConnect().setSQL("update contentlet "
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210719CleanUpDotAssetTitle.java:50]: 2 questions
1) why do not use prepared statements
2) why do not use batch updates

Comment (nollymar) [reply dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210719CleanUpDotAssetTitle.java]: Good suggestion!

Comment (nollymar) [reply dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210719CleanUpDotAssetTitle.java]: Done

**Review by jdotcms — APPROVED** [review]
great job

**Pull request closed without merging**